installer.py

```diff
@@ -1,3 +1,25 @@
+def full_install(package):
+    import configparser
```

Comment:
I didn't know this package, it's wonderfull

Reply:
It is very easy to use for the config.